eap/integration/eap7/src/test/resources/arquillian.xml

```diff
@@ -45,7 +45,7 @@
       environment, e.g. service host, port, etc. testrunner relates to the pod
       name in testrunner-pod.json.
     -->
-    <container qualifier="testrunner-eap7" mode="suite" default="true">
+    <container qualifier="testrunner-eap7" mode="class" default="true">
```

Comment:
I'm curious why this change was needed...

Reply:
This change makes the tests more stable.